### DISCUSSION TITLE
How Can I get Mobx Official SVG Icon and Color ?!

### DISCUSSION THREAD

**Discussion by Joylish** [mobxjs/mobx#2678] [Q&A]
Hi,I asked Simple Icon to create a mobx icon. [Simple-Icons](https://github.com/simple-icons)  is related to the [Shields](https://shields.io) badge.

[They made the following request to me.](https://github.com/simple-icons/simple-icons/issues/4352)

> Source: Unfortunately I haven't found any vectorized version of the Mobx logo. Neither on their website nor in their GitHub repo. Only [this PNG version](https://mobx.js.org/img/mobx.png).
> Color: I'm not sure which one to use. They are using different gradients of orange for their logo.

If you know, please answer.

**Comment by mweststrate** [comment]
Probably
https://github.com/mobxjs/mobx/blob/main/docs/assets/mobservable.svg
